code/modules/paperwork/paper.dm

```diff
@@ -304,16 +300,10 @@
 
 
 /obj/item/paper/fire_act(exposed_temperature, exposed_volume)
-	..()
+	. = ..()
 	if(!(resistance_flags & FIRE_PROOF))
```

Comment:
This could probably use check for ON_FIRE so it doesn't do the text processing multiple times.

Reply:
Honestly I left it cause I like the idea of fire continuing to make it hard to read the message as it continues.